assets/user-manifests-bootstrapper/user-manifests-bootstrapper-pod.yaml

```diff
@@ -21,6 +21,10 @@ kind: Pod
 metadata:
   name: manifests-bootstrapper
 spec:
+{{- if .ManifestBootstrapperSecurityContext.RunAsGroup }}
+  securityContext:
+    fsGroup: {{ .ManifestBootstrapperSecurityContext.RunAsGroup }}
```

Comment:
The fsGroup is different than the run as group.  We should probably discuss why we need to use fsGroup rather than ensuring all containers have access to the files via their run as group.

Reply:
Based on our testing, the cluster version operator container needs to create additional directories properly in order to run non-root.